lib/web.ts

```diff
@@ -43,7 +43,7 @@ export default class AutoConsent {
     };
     domActions: DomActions;
     filtersEngine: FiltersEngine;
-    protected sendContentMessage: MessageSender;
+    sendContentMessage: MessageSender;
```

Comment:
Maybe `_runRulesSequentially` in `AutoConsentCMP` should throw an error that `AutoConsent` catches? We're mixing concerns somewhat by having `AutoConsentCMP` know about the messaging channel.